lib/plugins/inspec-reporter-junit/README.md

```diff
@@ -10,6 +10,10 @@ This plugin is included with inspec. There is no need to install it separately.
 
 This reporter generates an XML report in Apache Ant JUnit format.
 
+'junit' is the legacy Chef InSpec JUnit reporter. It generates nonstandard JUNit XML in several ways. New users are advised to use junit2.
```

Comment:
I feel this documentation could be improved. But since we're addressing a regression with a merge fix we shouldn't hold up on merging - I can raise a follow-up PR with some suggested changes and assign to docs team.